src/install.rs

```diff
@@ -502,7 +502,19 @@ fn stash_saved_partitions(disk: &mut File, saved: &SavedPartitions) -> Result<()
 fn is_dasd(config: &InstallConfig) -> Result<bool> {
     let target = canonicalize(&config.device)
         .chain_err(|| format!("getting absolute path to {}", config.device))?;
-    Ok(target.to_string_lossy().starts_with("/dev/dasd"))
+    if !target.to_string_lossy().starts_with("/dev/dasd") {
+        return Ok(false);
+    }
+    let name = target
+        .file_name()
+        .chain_err(|| format!("getting name of {}", target.display()))?
+        .to_string_lossy();
+    let devtype = format!("/sys/class/block/{}/device/devtype", name);
+    let devtype = std::fs::read_to_string(&devtype).chain_err(|| format!("reading {}", devtype))?;
+    if !devtype.starts_with("3390/") {
+        bail!("only ECKD-DASD (type 3390) devices are supported");
```

Comment:
Shouldn't we return `Ok(false)` here? Or does coreos-installer currently not work at all with FBA?

Reply:
Currently FBA is not supported - it cannot (and shoudn't ) be configured as ECKD. Regular installation fails as well, so for now it's better to beak installation at this point.